csm/cli/cli_validators.py

```diff
@@ -0,0 +1,62 @@
+# CORTX-CSM: CORTX Management web and CLI interface.
```

Comment:
The CLI validators class will be required for other components too. You can have a base class in py-utils having commons validations and extend that class in CSM for CSM specific validations.